docs/get-started/getting-started-databricks.md

```diff
@@ -42,17 +32,17 @@ when using the plugin. Queries may still see significant speedups even with AQE
    
    See [issue-3143](https://github.com/NVIDIA/spark-rapids/issues/3143) for more detail.
 
-3. When selecting GPU nodes, Databricks requires the driver node to be a GPU node.  Outside of
+2. When selecting GPU nodes, Databricks requires the driver node to be a GPU node.  Outside of
```

Comment:
Do we want to comment that this limitation is with the Databricks UI, but the API interface allows selecting CPU driver nodes?